pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -949,11 +949,11 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
       // Sort by stage, question, result text.
       surveyResults.sort(function (a, b) {
         if (a.stage !== b.stage) {
-          return a.stage - b.stage;
+          return a.stage.localeCompare(b.stage);
```

Comment:
Discussed offline, these are only for courses that are English only so won't be used in other locales.